client-js/app/elements/labrad-registry.ts

```diff
@@ -206,34 +207,41 @@ export class LabradRegistry extends polymer.Base {
     if (event.ctrlKey || event.button == 2 ) {
```

Comment:
nvm; after digging a bit more it seems that support for `dropEffect` is woefully inconsistent across different browsers, with chrome being a particular offender. So looks like that's not really something we can rely on.